applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -250,7 +244,7 @@ int ext_sensors_mov_thres_set(double threshold_new)
 	if (err) {
 		LOG_ERR("Failed to set accelerometer y-axis threshold value");
 		LOG_ERR("Device: %s, error: %d",
-			accel_sensor.dev_name, err);
+			temp_sensor.dev->name, err);
```

Comment:
Should use device name for accelerometer.

Reply:
Fixed